src/PostgREST/DbStructure.hs

```diff
@@ -63,12 +63,12 @@ import Protolude.Unsafe (unsafeHead)
 
 
 data DbStructure = DbStructure
-  { dbTables      :: [Table]
-  , dbColumns     :: [Column]
-  , dbRelations   :: [Relation]
-  , dbPrimaryKeys :: [PrimaryKey]
-  , dbProcs       :: ProcsMap
-  , pgVersion     :: PgVersion
+  { dbTables        :: [Table]
+  , dbColumns       :: [Column]
+  , dbRelationships :: [Relation]
```

Comment:
What's the reason for that change? If it was my suggestion to not use abbreviations for field names: "Relation" is not an abbreviation. It's a full word. And our type is named Relation, too, so I'd leave it at that!

Reply:
Ah, the thing is that Relation means a table or view under pg(also under Relational algebra, I believe) and I thought the term was confusing. The "Relationship" name is more fit in this case. This is currently under a comment:

```haskell
-- | "Relation"ship between two tables.
--
data Relation = Relation
```

The `dbRelationships` would actually show on the resulting json schema cache, so that's why I only changed that. But you're right, I can leave this unrelated change out of this PR.

Besides that, WDYT about renaming Relation to Relationship?

Reply:
> Besides that, WDYT about renaming Relation to Relationship?

Agree to not use Relation, it's confusing in the relational db context!

Reply:
Absolutely agree! I was just confused by the inconsistency. If we change it, which we should, we should change it everywhere - especially on the type itself.

Reply:
Cool, done!